wellindexcalculator.cpp

```diff
@@ -150,7 +150,7 @@ void WellIndexCalculator::collect_intersected_cells(vector<IntersectedCell> &isc
     }
 
     // First cell
-    double epsilon = smallest_grid_cell_dimension_ / (100.0 * (start_pt-end_pt).norm());
+    double epsilon = smallest_grid_cell_dimension_ / (1e1 * (start_pt-end_pt).norm());
```

Comment:
Good question. Looks like a mistake in one of the reruns. Will rerun to be sure. Comment on new settings for reruns in Scratch discussion.